frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,88 @@
+   And user can see Vulnerable Container Images and Fixable Container Images
+   And user can see View all link on bottom of pop over
+
+@regression
+Scenario: Visit Image Manifest Vulnerabilities page from status in project overview    
+   Given user is at project tab
+   When user clicks on the Image Vulnerabilities in Status section
+   And user clicks on the View all link on Image Vulnerability breakdown pop over
+   Then user can see Image Manifest Vulnerabilities page
+
+@regression
+Scenario: Visit ImageManifestVuln from status in project overview    
+   Given user is at project tab
+   When user clicks on the Image Vulnerabilities in Status section
+   And user clicks on the image name in Image Vulnerability breakdown pop over to open that particular ImageManifestVuln
+   Then user can see ImageManifestVuln for the image name
+
+@regression
+Scenario: Visit quay page from status in project overview    
+   Given user is at project tab
+   When user clicks on the Image Vulnerabilities in Status section
+   And user clicks on the fixable link in Image Vulnerability breakdown pop over
+   Then user can see the image open in quay.io
+
+@regression
+Scenario: Vulnerability tab in developer console    
+   Given user is at topology page
+   When user goes to project tab on navigation menu
+   And user clicks on the vulnerability tab
+   Then user can see Image Manifest vulnerabilities
+   And user can see filter with name option selected
+   And user can see Image Name, Highest Severity, Affected Pods, Fixable, Total, and Manifests
+   
+@regression
+Scenario: Filter in vulnerability tab
+   Given user is at vulnerability tab
+   When user clicks on dropdown associated with filter
+   And user clicks on label option
+   Then user can see filter with label option selected
+
+@regression
+Scenario: Image manifests vulnerability details page
+   Given user is at vulnerability tab
+   And vulnerability tab has vulnerabilities
+   When user clicks on first vulnerability to open Image manifests vulnerability details page
+   Then user can see image manifest vulnerability Details, YAML and Affected Pods tabs on ImageManifestsVuln Details page
+   And user can see the donut chart ImageManifestsVuln section
+   And user can see All vulnerabilities selected for donut chart
+   And user can see App dependency and Base image options available for donut chart
+   And user can see vulnerabilities at the bottom of the page
+   And user can see filter in vulnerabilities section
+   And user can see search by name input field
+
+@regression
+Scenario: Filters of Image manifests vuln section in Image manifests vulnerability details page
+   Given user is at Image manifests vulnerability details page
+   When user clicks on App dependency filter in donut chart
+   Then user can see vulnerabilities due to app dependency in donut chart
+   And user clicks on Base image filter in donut chart
+   And user can see vulnerabilities due to Base image in donut chart
```

Comment:
I think it's a separate scenario - You are trying to verify base filter, please update it